src/tz_macos.rs

```diff
@@ -5,27 +5,37 @@ use core_foundation_sys::string::{
 use core_foundation_sys::timezone::{CFTimeZoneCopySystem, CFTimeZoneGetName};
 
 pub(crate) fn get_timezone_inner() -> Result<String, crate::GetTimezoneError> {
-    unsafe { get_timezone().ok_or(crate::GetTimezoneError::OsError) }
+    get_timezone().ok_or(crate::GetTimezoneError::OsError)
 }
 
 #[inline]
-unsafe fn get_timezone() -> Option<String> {
+fn get_timezone() -> Option<String> {
     // The longest name in the IANA time zone database is 25 ASCII characters long.
     const MAX_LEN: usize = 32;
 
     // Get system time zone, and borrow its name.
-    let tz = Dropping::new(CFTimeZoneCopySystem())?;
-    let name = CFTimeZoneGetName(tz.0);
+    let tz = Dropping::new(unsafe { CFTimeZoneCopySystem() })?;
+    let name = unsafe { CFTimeZoneGetName(tz.0) };
```

Comment:
```diff
+// SAFETY: `Dropping::new()` would have returned `None` if the call to `CFTimeZoneCopySystem()` had failed.
```

Reply:
👍 to @Kijewski. Let's add safety comments to these unsafe blocks. A safety comment should document how the code is upholding the invariants of the unsafe functions being called.